generators/client/templates/angular/src/test/javascript/e2e/admin/administration.spec.ts.ejs

```diff
@@ -0,0 +1,101 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+import { browser, element, by, ExpectedConditions as ec } from 'protractor';
+
+import { NavBarPage, SignInPage } from '../page-objects/jhi-page-objects';
+<%_
+let elementGetter = `getText()`;
+if (enableTranslation) {
+    elementGetter = `getAttribute('jhiTranslate')`;
+}
+_%>
+
+const expect = chai.expect;
+
+describe('administration', () => {
+    let navBarPage: NavBarPage;
+    let signInPage: SignInPage;
+
+    before(async () => {
+        await browser.get('/');
+        navBarPage = new NavBarPage(true);
+        signInPage = await navBarPage.getSignInPage();
+        <%_ if (authenticationType !== 'oauth2') { _%>
+        await signInPage.autoSignInUsing('admin', 'admin');
+        <%_ } else { _%>
+        await signInPage.loginWithOAuth('admin', 'admin');
```

Comment:
These should be changed so they can be overridden by environment variables. This was a recent change in the main generator.

```
const username = process.env.E2E_USERNAME || 'admin';
const password = process.env.E2E_PASSWORD || 'admin';
```

See https://github.com/jhipster/generator-jhipster/pull/11448 for more info.

Reply:
@mraible Thanks! I made the same change in the modified tests.